docs/refs.bib

```diff
@@ -317,6 +328,18 @@ @misc{Cariello_2013_Weak_irreducible
 
 
 % Last name begins with D
+
+@article{Doherty2004_CompleteFamily,
+  author = {Doherty, Andrew C. and Parrilo, Pablo A. and Spedalieri, Federico M.},
+  doi = {10.1103/PhysRevA.69.022308},
+  journal = {Physical Review A},
+  number = {2},
+  pages = {022308},
+  title = {Complete Family of Separability Criteria},
+  volume = {69},
+  year = {2004}
+}
```

Comment:
Entries are arranged alphabetically in this file. Please move this entry to after L369